frontend/src/features/documents/hooks/useDocumentRelationshipProvider.ts

```diff
@@ -46,16 +46,24 @@ export const useDocumentRelationshipProvider = () => {
   } = useApiRequestWrapper<
     (
       relationshipType: DocumentRelationshipType,
-      parentId: number,
+      documentRelationship: Api_DocumentRelationship,
     ) => Promise<AxiosResponse<boolean, any>>
   >({
     requestFunction: useCallback(
-      async (relationshipType: DocumentRelationshipType, parentId: number) =>
-        await deleteDocumentRelationshipApiCall(relationshipType, parentId),
+      async (
+        relationshipType: DocumentRelationshipType,
+        documentRelationship: Api_DocumentRelationship,
+      ) => await deleteDocumentRelationshipApiCall(relationshipType, documentRelationship),
       [deleteDocumentRelationshipApiCall],
     ),
     requestName: 'deleteDocumentRelationship',
-    onSuccess: useCallback(() => toast.success('Delete document relationship'), []),
+    onSuccess: useCallback((response?: boolean) => {
+      if (response !== undefined && response) {
+        toast.success('Delete document relationship');
+      } else {
+        toast.error('Delete document relationship error. Check responses and try again.');
+      }
+    }, []),
```

Comment:
question to learn - I'm a but confused by this... so the API could return 200 and still be an error?

Reply:
Yes, that would mean that the call to the external API was successful (httpstatus 200 /202) but the action was not what we wanted (i.e. not found).